tests/rptest/tests/acls_test.py

```diff
@@ -321,6 +321,91 @@ def test_mtls_principal(self, rules=None, fail=False):
             except ClusterAuthorizationError:
                 assert fail, "list acls should have succeeded"
 
+    @cluster(num_nodes=3)
+    @parametrize(authn_method='sasl')
+    @parametrize(authn_method='mtls_identity')
+    def test_security_feature_migration(self, authn_method: str):
+        def check_perms():
```

Comment:
This code is duplicated in a few places in this file. Could we parametrise this method and lift it to get rid of the duplication?

Reply:
I get mixed reviews about generalizing test code in PRs. Some folks say to generalize things other folks say to keep the duplication because generalizing increasing the amount of info that readers must keep in their head.

I'll get rid of the duplication for now though.

Reply:
Done.